### ISSUE TITLE
TFX compatibility issue Op type not registered 'RegexSplitWithOffsets' in binary

### ISSUE THREAD

**Issue by tommywei110** [tensorflow/text#311]

**Comment by tommywei110** [comment]
Versions:
Tensorflow: 2.2.0
tfx: 0.22
tensorflow_text: 2.2.1

`RuntimeError: tensorflow.python.framework.errors_impl.NotFoundError: Op type not registered 'RegexSplitWithOffsets' in binary running on penguin. Make sure the Op and Kernel are registered in the binary running in this process. Note that if you are loading a saved graph which used ops from tf.contrib, accessing (e.g.) `tf.contrib.resampler` should be done before importing the graph, as contrib ops are lazily registered when the module is first accessed. [while running 'Analyze/ComputeDeferredMetadata']
 [while running 'Run[Transform]']`

I encounter this error while using `tensorflow_text.BertTokenizer` in the `preprocess_fn()` of tfx transform component. 
[Link to tfx.transformer](https://www.tensorflow.org/tfx/guide/transform#the_tensorflow_transform_preprocessing_fn)

Code to reproduce:
`def preprocessing_fn(inputs):
  tokenizer = text.BertTokenizer('vocab.txt')
  output = tokenizer.tokenize(inputs['param'])
`

While testing, we found the same version settings on colab would have no problem.
Executing the tokenize step in an eager execution in local environment would also work.
i.e
`
if __name__ == '__name__':
  tokenizer = text.BertTokenizer('vocab.txt')
  output = tokenizer.tokenize(inputs['param'])
`

So the hypothesis is when TFX constructs the graph for deferred execution, some binary ops are not properly compiled/imported.

**Comment by broken** [comment]
#315 has more info with an example

**Comment by krystollia** [comment]
#261 is the same issue, hope this will be solved soon 🙁

**Comment by broken** [comment]
TF.Transform has addressed this on their end (https://github.com/tensorflow/transform/commit/7ec6244fffbcce76e38f0a59472f3a0b282e34ef). This should be fixed in their next release.
